middleware/correlation.go

```diff
@@ -3,21 +3,26 @@ package middleware
 import (
 	"context"
 	"net/http"
+	"regexp"
 
 	cs "github.com/cloudtrust/common-service"
 	gen "github.com/cloudtrust/common-service/idgenerator"
 	"github.com/cloudtrust/common-service/log"
 	"github.com/cloudtrust/common-service/tracing"
 )
 
+const (
+	regExpCorrelationID = `^[\w\d_#@-]{1,70}$`
```

Comment:
Could we have a correlation ID of size 70? Could we set the max as the normal size + margin (e.g. max 10 characters extra) ?

Reply:
There is no exact max length... The first part has not a fixed length as it is the component name. I used the length of the sample from the unit tests and added 10

Reply:
Alright then, ok for me